APE24.rst

```diff
@@ -0,0 +1,507 @@
+|                          | is still maintained and in  | updated.                   |
+|                          | what state. We might be able|                            |
+|                          | to work with Astropy (and   |                            |
+|                          | other communities) on what  |                            |
+|                          | this looks like.            |                            |
++--------------------------+-----------------------------+----------------------------+
+
+.. _pyOpenSci Editorial Board: https://www.pyopensci.org/about-peer-review/#our-editorial-board
+.. _Astropy team roles: https://www.astropy.org/team#roles
+
+As of July 2023, Astropy had the following `pre-APE 24 Affiliated Package review guidelines`_
+and followed this `pre-APE 24 Affiliated Package review process, with templates`_.
+
+.. _pre-APE 24 Affiliated Package review guidelines: https://github.com/astropy/astropy-project/blob/1afe5df1728e86dac51b9b87bc30a7c2cdb289ac/affiliated/affiliated_package_review_guidelines.md
+.. _pre-APE 24 Affiliated Package review process, with templates: https://github.com/astropy/astropy-project/blob/1afe5df1728e86dac51b9b87bc30a7c2cdb289ac/affiliated/affiliated_package_review_process.md
+
+Meanwhile, pyOpenSci has its own
+`submission guide for package authors <https://www.pyopensci.org/software-peer-review/how-to/author-guide.html>`_,
+`accompanying GitHub issue templates <https://github.com/pyOpenSci/software-submission/issues/new/choose>`_, and
+`listing of accepted packages <https://www.pyopensci.org/python-packages.html#our-accepted-python-open-source-packages>`_.
+As of July 2023, they are in the process of revamping their process to handle
+collaborations with different communities (e.g., Astropy). They are also:
+
+* creating a community-specific label for package filtering
+  for that community on their website package listing;
+* updating the submission template so a potential Astropy affiliated package
+  can select Astropy for the actual review;
+* adding an Astropy label on their GitHub repository; and
+* creating a new editorial bot to add community-specific tags.
+
+Is open review a deal breaker?
+------------------------------
+
+We do not think so! In the pre-APE 24 process, the reviewer was anonymous by default
+because we modeled the process after traditional astronomical journals.
+Even then, the reviewer had the option to reveal themselves if they wanted to.
+Given that the Astropy ecosystem is open-source anyway, it makes sense for us
+to move away from the journal-style tradition and embrace a more open process that
+pyOpenSci uses.
+
+Will we no longer need Editors?
+-------------------------------
+
+We still need them! However, instead of managing the whole process themselves,
+they will now be part of the `pyOpenSci Editorial Board`_ and perform their
+duties within the pyOpenSci process as laid out in this APE.
```

Comment:
This means the role will remain in the astropy roles list, is that correct?  Just also listed in the pyOpenSci process?

(A roles change technically requires additional steps, so I just want it to be explicit if so, but my read on this is that it is *not* the case.  Just wanting clarification.)

Reply:
Same role, different tasks, is how I view it. We definitely still need our own Editors but they would work within pyOpenSci process as laid out in this APE.

Reply:
So in practice, this role stays on Team page. pyOpenSci will have these same people listed on their Editors page and maybe even add Astropy badge to their profiles (@lwasser , please confirm).

Reply:
Perfect!  All good there then.